core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -840,6 +844,8 @@ private SqlCharStringLiteral makeRegexNode(SqlCall call) {
     case OTHER_FUNCTION:
       unparseOtherFunction(writer, call.operand(1), leftPrec, rightPrec);
       break;
+    case PLUS:
```

Comment:
Explain the change.

Reply:
I wrote a addition logic at mig side ,earlier it was failing with below exception 
error :  plus is not valid
after adding 'plus' case in switch statement it will return statement and will not throw any error .